app/users/resources.py

```diff
@@ -0,0 +1,95 @@
+import json
+
+from flask import request
+from flask_restful import Resource
+
+from app import db, User
+from app.lib.response_util import buildOkResponse
+
+
+def returnUser(user):
+    """Private method to convert a user model into a dictionary.
+
+        Args:
+        user: A user model.
```

Comment:
nit: Please indent the user arg over one tab and separate the description with a dash.
```
"""Private method to convert user model into a dictionary.

Args:
    user - A user database model.
"""
```